test/items_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'CSV'
```

Comment:
Do we need to require CSV here? If we're creating our own hash for the test?

Reply:
I think you're right, when we were making the test initially I assumed we might need it so I told him to throw it in, looks like we don't!